include/rocksdb/statistics.h

```diff
@@ -640,11 +632,15 @@ enum Histograms : uint32_t {
   FLUSH_TIME,
   SST_BATCH_SIZE,
 
+  // Number of IOs issued in parallel in a MultiGet batch
```

Comment:
`SST_IO_BATCH_SIZE` is missing according to the test failure

Reply:
Thanks for the heads up. Fixed it.